.github/workflows/playwright.yml

```diff
@@ -0,0 +1,27 @@
+# name: Playwright Tests
```

Comment:
I created a card to deal with this => https://trello.com/c/7pmq4H7E/19-to-do-app-setup-github-actions-to-run-playwright-tests-in-pr-open

Reply:
👍🏼